mw-webapp/src/component/tooltip/Tooltip.module.scss

```diff
@@ -35,14 +33,19 @@ th .wrapper {
 }
 
 .right {
-  margin-top: -$marginSmall;
-  transform: translateX(100%);
+  margin-left: $tooltipMarginLeft;
+  transform: translateY(-100%);
 }
```

Comment:
why do you need both translater and margin

Reply:
Deleted margin